bin/create-hydra-client.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console */
+const http = require("http");
+
+/* eslint-disable camelcase */
+const bodyEncoded = JSON.stringify({
+  client_id: process.env.OAUTH2_CLIENT_ID || "reaction-next-starterkit",
+  client_secret: process.env.OAUTH2_CLIENT_SECRET || "CHANGEME",
```

Comment:
Don't do fallbacks in here. This should throw / exit non-zero if any of the necessary env variables are missing.

Reply:
OK I don't see any validation of env vars in the main app either. Do we have a library we could use for schema checking and just run that against `process.env`? Only thing I could find was `react-props` but that seems to not advise running outside of React.